src/applications/mhv/landing-page/hooks/useDatadogRum.jsx

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from 'react';
+import { datadogRum } from '@datadog/browser-rum';
+
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+
+const initializeDatadogRum = () => {
+  if (
+    // Prevent RUM from running on local/CI environments.
+    environment.BASE_URL.indexOf('localhost') < 0 &&
+    // Prevent re-initializing the SDK.
+    !window.DD_RUM?.getInitConfiguration()
+  ) {
+    datadogRum.init({
+      applicationId: '1f81f762-c3fc-48c1-89d5-09d9236e340d',
+      clientToken: 'pub3e48a5b97661792510e69581b3b272d1',
+      site: 'ddog-gov.com',
+      service: 'mhv-on-va.gov',
+      env: environment.vspEnvironment(),
+      sampleRate: 100,
+      sessionReplaySampleRate: 0,
+      trackInteractions: true,
+      trackResources: true,
+      trackLongTasks: true,
+      defaultPrivacyLevel: 'mask-user-input',
+    });
+  }
+};
+
+const useDatadogRum = () => {
+  useEffect(() => {
+    initializeDatadogRum();
+  }, []);
```

Comment:
I noticed that in another implementation, their useEffect depends on two feature flags. Are those site wide flags we should consider?

 https://github.com/department-of-veterans-affairs/vets-website/blob/dc303fe336ec8a0717fe7e259d91d5e302067d89/src/applications/check-in/hooks/useDatadogRum.jsx#L47

Reply:
`isBrowserMonitoringEnabled` is a feature flag for the other implementation 

`isLoadingFeatureFlags` is the feature flag endpoint API loading status. 

Neither of which we care about.